README.md

```diff
@@ -2,12 +2,8 @@
 A sublime plugin for [Metals](https://scalameta.org/metals/) proving status bar support and other custom Metals LSP endpoints.
 
 # Installation and usage
-1. install sublime [LSP](https://github.com/tomv564/LSP) plugin
-2. Clone the project into the Packages directory of sublime. 
-```
-# macOS
-~/Library/Application\ Support/Sublime\ Text\ 3/Packages/
-# Ubuntu
-~/.config/sublime-text-3/Packages/
-```
-3. Enable "metals-sublime" in your scala project
+1. Install sublime [LSP](https://github.com/tomv564/LSP) plugin
+2. In sublime's command palette type `Package Control: Add Repository`
+3. At the bottom of the Sublime window, an input box will appear. Paste the URL of this repository (https://github.com/scalameta/metals-sublime)
+4. In sublime's command palette type `Package Control: Install Packages` and select `metals-sublime`
+5. Finally enable "metals-sublime" in your scala project via the command palette
```

Comment:
It was not clear to me what command to invoke here, is this correct?

<img width="626" alt="Screenshot 2019-11-17 at 05 13 23" src="https://user-images.githubusercontent.com/1408093/69003386-0ad79180-08f9-11ea-901e-bef2a492055a.png">

Reply:
I updated the plugin behavior and the doc.
It should be enabled by default now on all scala projects so one step less for the user :)
I guess it is the same thing with the vscode plugin